features/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/internal/recorder/base64/WebPImageCompressionTest.kt

```diff
@@ -85,5 +86,17 @@ internal class WebPImageCompressionTest {
         assertThat(captor.firstValue).isEqualTo(Bitmap.CompressFormat.WEBP)
     }
 
+    @Test
+    fun `M return empty bytearray W compressBitmap { bitmap was already recycled }`() {
+        // Given
+        whenever(mockBitmap.isRecycled).thenReturn(true)
+
+        // When
+        val result = testedImageCompression.compressBitmap(mockBitmap)
+
+        // Then
+        assertThat(result).isEqualTo(ByteArray(0))
```

Comment:
probably `isEmpty` assertion will be more descriptive here

```suggestion
        assertThat(result).isEmpty()
```